# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Various fixes for all French locales
# DP: Related bugs: #248377  #351786 #345481
# DP: Dpatch author: Denis Barbier <barbier@debian.org>
# DP: Patch author: Denis Barbier
# DP: Upstream status: not submitted
# DP: Date: 2006-01-10

---
 localedata/locales/fr_BE |    9 +++++++--
 localedata/locales/fr_CA |    4 ++++
 localedata/locales/fr_CH |    9 +++++++--
 localedata/locales/fr_FR |   11 +++++++++++
 localedata/locales/fr_LU |    9 +++++++--
 5 files changed, 36 insertions(+), 6 deletions(-)

--- a/localedata/locales/fr_BE
+++ b/localedata/locales/fr_BE
@@ -124,6 +124,10 @@
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
+date_fmt "<U0025><U0041><U0020><U0025><U002D><U0065><U0020>/
+<U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
+<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
+<U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
 
 week    7;19971130;4
 first_weekday 2
@@ -145,8 +149,7 @@
 END LC_MEASUREMENT
 
 LC_NAME
-name_fmt    "<U0025><U0064><U0025><U0074><U0025><U0067><U0025><U0074>/
-<U0025><U006D><U0025><U0074><U0025><U0066>"
+copy "fr_FR"
 END LC_NAME
 
 LC_ADDRESS
@@ -159,6 +162,8 @@
 country_ab2 "<U0042><U0045>"
 country_ab3 "<U0042><U0045><U004C>"
 country_num 056
+country_post "<U0042><U0045>"
+country_isbn 2
 % B
 country_car    "<U0042>"
 % Français
--- a/localedata/locales/fr_CA
+++ b/localedata/locales/fr_CA
@@ -119,6 +119,10 @@
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
+date_fmt "<U0025><U0041><U0020><U0025><U002D><U0065><U0020>/
+<U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
+<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
+<U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
 week 7;19971130;1
 END LC_TIME
 
--- a/localedata/locales/fr_CH
+++ b/localedata/locales/fr_CH
@@ -108,6 +108,10 @@
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
+date_fmt "<U0025><U0041><U0020><U0025><U002D><U0065><U0020>/
+<U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
+<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
+<U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
 week 7;19971130;4
 first_weekday 2
 END LC_TIME
@@ -127,8 +131,7 @@
 END LC_MEASUREMENT
 
 LC_NAME
-name_fmt    "<U0025><U0064><U0025><U0074><U0025><U0067><U0025><U0074>/
-<U0025><U006D><U0025><U0074><U0025><U0066>"
+copy "fr_FR"
 END LC_NAME
 
 LC_ADDRESS
@@ -141,6 +144,8 @@
 country_ab2 "<U0043><U0048>"
 country_ab3 "<U0043><U0048><U0045>"
 country_num 756
+country_post "<U0043><U0048>"
+country_isbn 2
 % CH
 country_car    "<U0043><U0048>"
 % Français
--- a/localedata/locales/fr_FR
+++ b/localedata/locales/fr_FR
@@ -143,6 +143,10 @@
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
+date_fmt "<U0025><U0041><U0020><U0025><U002D><U0065><U0020>/
+<U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
+<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
+<U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
 
 week    7;19971130;4
 first_weekday 2
@@ -167,6 +171,11 @@
 LC_NAME
 name_fmt    "<U0025><U0064><U0025><U0074><U0025><U0067><U0025><U0074>/
 <U0025><U006D><U0025><U0074><U0025><U0066>"
+name_gen    ""
+name_mr     "<U004D><U002E>"
+name_mrs    "<U004D><U006D><U0065>"
+name_miss   "<U004D><U006C><U006C><U0065>"
+name_ms     ""
 END LC_NAME
 
 LC_ADDRESS
@@ -179,6 +188,8 @@
 country_ab2 "<U0046><U0052>"
 country_ab3 "<U0046><U0052><U0041>"
 country_num 250
+country_post "<U0046><U0052>"
+country_isbn 2
 % F
 country_car    "<U0046>"
 % Français
--- a/localedata/locales/fr_LU
+++ b/localedata/locales/fr_LU
@@ -124,6 +124,10 @@
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
+date_fmt "<U0025><U0041><U0020><U0025><U002D><U0065><U0020>/
+<U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
+<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
+<U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
 week 7;19971130;4
 first_weekday 2
 END LC_TIME
@@ -143,8 +147,7 @@
 END LC_MEASUREMENT
 
 LC_NAME
-name_fmt    "<U0025><U0064><U0025><U0074><U0025><U0067><U0025><U0074>/
-<U0025><U006D><U0025><U0074><U0025><U0066>"
+copy "fr_FR"
 END LC_NAME
 
 LC_ADDRESS
@@ -157,6 +160,8 @@
 country_ab2 "<U004C><U0055>"
 country_ab3 "<U004C><U0055><U0058>"
 country_num 442
+country_post "<U004C><U0055>"
+country_isbn 2
 % L
 country_car    "<U004C>"
 % Français
